src/RelayProvider.ts

```diff
@@ -237,6 +242,73 @@ export default class RelayProvider implements HttpProvider {
         return this.executeRelayTransaction(transactionDetails);
     }
 
+    async processTransactionReceipt(
```

Comment:
probably this method should be private, shouldn't it?

Reply:
done